grafana-dashboard/ClickHouse_Queries_dashboard.json

```diff
@@ -870,21 +885,65 @@
     "list": [
       {
         "current": {
-          "text": "k8s-test-test-001",
-          "value": "k8s-test-test-001"
+          "text": "k8s-test-test-002-tpl",
+          "value": "k8s-test-test-002-tpl"
         },
-        "hide": 0,
+        "hide": 2,
         "includeAll": false,
         "label": "K8S ClickHouse installation",
         "multi": false,
-        "name": "chi",
+        "name": "db",
         "options": [],
         "query": "vertamedia-clickhouse-datasource",
         "refresh": 1,
-        "regex": "/k8s/",
+        "regex": "k8s-$exported_namespace-$chi",
```

Comment:
main part of fix
add dependency on prometheus label related variables
@amonakhov so created clickhouse cluster datasource, should follow this naming conversion